### ISSUE TITLE
Website requires some love

### ISSUE THREAD

**Issue by SHAESEN2** [openpharma/visR#333]
1) Group functions together in the reference section
2) Add a section for the datasets
3) After #336 and #339 have been merged, it'd be cool to split our vignettes by "static" and "interactive"

**Comment by timtreis** [comment]
Re 3: After merging we'll automatically be deploying a learnR vignette we've showcased at workshops. However, this vignette spans several topics. It'd be cool to split it out into individual workflows/case-studies as well.
